src/state/transactions/updater.tsx

```diff
@@ -15,6 +16,23 @@ interface TxInterface {
   lastCheckedBlockNumber?: number
 }
 
+export function checkPrivateTxStatusFailed(txHash: string): Promise<boolean> {
+  return new Promise((resolve) => {
```

Comment:
we typically use `rtk-query` over fetching manually

https://redux-toolkit.js.org/rtk-query/overview

would involve creating a new api slice to interact with `api.flashbots.net`, similar to https://github.com/Uniswap/interface/blob/main/src/state/data/slice.ts